app/controllers/appointments_bookings_controller.rb

```diff
@@ -9,12 +9,8 @@ def load_places
     # Load places
     convict_places = Place.kept.joins(:appointment_types, organization: :convicts)
                           .where('convicts.id': @convict.id).where(appointment_types: @appointment_type)
-    # Les agents SAP doivent pouvoir prendre des convocations SAP DDSE au SPIP
-    user_places = if current_user.work_at_sap? && @appointment_type.ddse?
-                    Place.in_jurisdiction(current_user.organization)
-                  else
-                    policy_scope(Place).kept
-                  end
+
+    user_places = policy_scope(Place).joins(:appointment_types, organization: :convicts).kept
```

Comment:
Tu as besoin des joins ici ?

Reply:
a priori oui sinon erreur de "and" mais maintenant qu'on sait qu'on ne peut pas faire confiance a ce genre d'erreur. Peut etre qu'il y a un truc sous-jacent